bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -5,18 +5,20 @@
 from typing import Union
 
 import discord
+from async_rediscache import RedisCache
 from discord import Color, Embed, Member, PartialMessage, RawReactionActionEvent, User
 from discord.ext.commands import Cog, Context, group, has_any_role
 
 from bot.api import ResponseCodeError
 from bot.bot import Bot
-from bot.constants import Channels, Emojis, Guild, MODERATION_ROLES, STAFF_ROLES, Webhooks
+from bot.constants import Channels, Emojis, Guild, MODERATION_ROLES, Roles, STAFF_ROLES, Webhooks
 from bot.converters import FetchedMember
 from bot.exts.moderation.watchchannels._watchchannel import WatchChannel
 from bot.exts.recruitment.talentpool._review import Reviewer
 from bot.pagination import LinePaginator
 from bot.utils import time
 
+AUTOREVIEW_ENABLED_KEY = 'autoreview_enabled'
```

Comment:
Python Discord uses double quotes (I won't bother with commenting on every single instance).

Reply:
True, although the cog (and the whole bot) is so mixed already that i'm not sure it's really worth it, imo if we want to enforce it it should be through a linting workflow, as suggested in https://github.com/python-discord/bot/issues/1641. I guess I can fix this on the bit's I PRd though, will leave the rest of the cog as-is though.